lib/Doctrine/Migrations/DbalMigrator.php

```diff
@@ -89,6 +91,15 @@ private function executeMigrations(
         return $sql;
     }
 
+    private function assertAllMigrationsAreTransactional(MigrationPlanList $migrationsPlan): void
+    {
+        foreach ($migrationsPlan->getItems() as $plan) {
+            if ($plan->getMigration()->isTransactional() === false) {
```

Comment:
No need for a strict comparison here, `isTransactional` is a boolean, that's enforced in the signature of the method, which means we can use the more expressive
```suggestion
            if (! $plan->getMigration()->isTransactional()) {
```